tox.ini

```diff
@@ -38,7 +38,7 @@ deps =
     numpy126: numpy==1.26.*
 
     astroscrappy11: astroscrappy==1.1.*
-    astroscrappy10: astroscrappy==1.0.*
```

Comment:
I'm also removing the astropyscrappy10 env as it doesn't seem to be used in CI, but obviously I'm happy to revert this if undesired.